File: 2020-Spring/COMSC-275 Intro to Web Programming/Assignments/16. Stock/PHP/searchStock.php
<?php

$symbol=$_GET['symbol'];

$format=$_GET['format'];

//reurns output in JSON format

if ($format=='json') {
    
    $s = file_get_contents(
        
        "https://query1.finance.yahoo.com/v7/finance/quote?symbols="
        
        . $symbol
                
        );
    
}

else{
    
    $s = "";
    
}

print $s;

?>

File: 2020-Spring/COMSC-275 Intro to Web Programming/Assignments/16. Stock/Python/searchStock.py
#!/Python27/python

import cgi

import requests

f=cgi.FieldStorage()
symbol=f.getvalue('symbol')
format=f.getvalue('format')

print("Content-type: text/txt\n\n")

#returns json
if format == 'json':

    r=requests.get(
    "https://query1.finance.yahoo.com/v7/finance/quote?symbols="+symbol)

    #status_code=r.status_code
    #s=(str(status_code))
    s=r.text

else:
    s = ""

print(s)